src/extra/AutoUi/Modals/SimpleConfirmationModal.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { ButtonProps } from '../../../components/Button';
+import { Modal } from '../../../components/Modal';
+import { useTranslation } from '../../../hooks/useTranslation';
+
+export interface SimpleConfirmationModalProps {
+	children: React.ReactNode;
+	title?: string | JSX.Element;
+	action?: string | JSX.Element;
+	primaryButtonProps?: ButtonProps;
+	onClose: (confirmed: boolean) => void;
+}
+
+export const SimpleConfirmationModal = (
```

Comment:
I think we already have this in `~/src/internal/SimpleConfirmationModal.tsx` right?
If so, then let's use that one.

Reply:
Nice catch thank you